### ISSUE TITLE
Troubles with Mac, can't set working directory

### ISSUE THREAD

**Issue by jojoman55** [BioEasy/EasyCodeML#2]
Hi 

I can't seem to get this to work with the example dataset. When I try to run it asks me to set the working directory first even though I have already done this .

Best,
Jojo

**Comment by BioEasy** [comment]
Hi Jojo,
Before starting a CodeML analysis, you have to press the button "Save Current Profile" to save parameter settings. 

All the best,
Fangluan
